pkg/envtest/helper.go

```diff
@@ -3,10 +3,21 @@ package envtest
 import (
 	"reflect"
 
+	apiextensionsinstall "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/install"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
+var (
+	crdScheme = runtime.NewScheme()
+)
+
+// init is required to correctly initialize the crdScheme package variable.
+// Installs the CRDv1 and CRDv1beta1 from the apiextensions package.
+func init() {
+	apiextensionsinstall.Install(crdScheme)
```

Comment:
prob better to explicitly add understood versions here

Reply:
It was explicit before, but I recommended to use this one-liner :/

Reply:
Any preferences here?

Reply:
Avoiding buzzwords, but here we are since I couldn't type it without them --

controller-runtime is an abstraction. We should use the highest form of abstraction that our dependencies afford us. I'm :+1: on using that version, and including that information in our release notes (as of this version of CR on Kubernetes 1.17, our API code targets CRDv1)

Reply:
@vincepri wdyt?

Reply:
To be a bit more concrete, too - I don't think CR tooling should bleed across the wire. Otherise, we have the API server, client-go, CR, and others all managing this concern -- which in this case should be determined by our pinned version of this dependency, not necessarily by us.

In a larger scope, this gets into minimal version selection (of which that given version, in our case CRD/v1beta1, will be eventually deprecated and removed). I'd lean toward choosing the "storage" version indicated by a given upstream dependency version, which I would assume that these helper methods/funcs would choose.

If they don't, that's when we override.